analyses/fusion_filtering/06-recurrent-fusions-per-histology.R

```diff
@@ -79,22 +84,15 @@ standardFusionCalls<-standardFusionCalls %>%
   left_join(clinical,by=c("Sample"="Kids_First_Biospecimen_ID","Kids_First_Participant_ID")) %>% 
   dplyr::filter(!is.na(broad_histology)) %>% as.data.frame()
 
-#remove fusions found in benign tumors as internal false positive control
-# KCNH1--AL590132.1 and AL590132.1--KCNH1 come up as recurrent in multiple histologies but using the following filter helps remove such fusions.
-
-# get the names of fusions that are present in benign tumors
-fusions_in_benign <- standardFusionCalls %>%
-  dplyr::filter(broad_histology == "Benign tumor") %>%
-  unique() %>%
-  dplyr::pull(FusionName)
+#remove fusions found in more than 4 histologies as internal false positive control
+# for example KCNH1--AL590132.1 and AL590132.1--KCNH1 come up as recurrent in multiple histologies are now being removed as potential false positive
```

Comment:
This happens upstream of this step at the moment, correct? If so, I would remove this because I would expect the code below to reflect this comment.